src/main/java/duke/main/DateTimeFormatUtils.java

```diff
@@ -0,0 +1,77 @@
+package duke.main;
+
+import duke.exception.DukeException;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Class encapsulating operations on LocalDateTime for task logging.
+ */
+public class DateTimeFormatUtils {
+    /* Formatter objects for valid input and output types */
+    private final static DateTimeFormatter INPUT_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
+    private final static DateTimeFormatter OUTPUT_FORMAT = DateTimeFormatter.ofPattern("MMM d yyyy HHmm");
+
+    /**
+     * Returns LocalDateTime object given String representing a date.
+     * Throws DukeException if input string does not match valid format.
+     *
+     * @param date String containing date information.
+     * @return LocalDateTime object retrieved from parsing input string.
+     * @throws DukeException If input string is of invalid formatting.
+     */
+    public static LocalDateTime parseDate(String date) throws DukeException {
+        try {
+            LocalDateTime inputDate = LocalDateTime.parse(date, INPUT_FORMAT);
+            return inputDate;
+        } catch (DateTimeParseException e) {
+            throw new DukeException("\tDate Format Police here!!" +
```

Comment:
Line break here should be before the operator.

Reply:
Hi, thanks for pointing this out! I really appreciate your help in reviewing and even including tips on using the IDE formatting settings! 😄 💯 👌